backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -50,12 +57,29 @@ public void setUp(){
         inMemoryVirtualQueueRepository.getShopOwnerMap().clear();
     }
 
+    private UUID addShopToRepository(){
+        Shop shop = new Shop(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
+        shop.setStatus(ShopStatus.ACTIVE);
```

Comment:
Can these 2 lines be merged?

Reply:
With our current design, don't think so.